### DISCUSSION TITLE
Create a Branch from a Tag Feature

### DISCUSSION THREAD

**Discussion by dinakarapra** [community/community#39285] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice to have feature that allows us to create a new branch from the tag in the "Create New Branch" section. The current alternative is to create a new branch from a tag locally and push the branch upstream. this requires considerable git knowledge and expreience.  

Currently "Create New Branch" feature only allows us to create a new branch only from another branch (main/master), but not from an existing tag. There many usecases where we may have to create a branch from a tag and work on. So it would be great addition to "Create New Branch" feature to be able to create a new branch out of an existing tag.

**Comment by LocatelliTonyAllPhi** [comment]
I fully agree! It's really sad not to have this possibility, it would be useful in a lot of cases (for hotfixes, to regenerate a nuget deleted by mistake, etc.).
